.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .auth.json
 _cache.db
 all*.json
+config.json
```

Comment:
Not sure why this is necessary, or why you would even create this file. You can run against the configs in-place: `./bin/sync.js projects/nodeops.json`